### PR TITLE
Update `Rating` JSDocs to remove ToDos

### PR DESCRIPTION
**Pull request by tomi-msft**
This PR is to finish the documentation of `Rating`. Tracked with #30652

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/6bd8fa75a19551ee7ac968d1619b49d8c10dbec8'>6bd8fa75a19551ee7ac968d1619b49d8c10dbec8</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

File: packages/react-components/react-rating-preview/src/components/Rating/Rating.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ import type { RatingProps } from './Rating.types';
 import { useRatingContextValues } from './useRatingContextValues';
 
 /**
- * Rating component - TODO: add more docs
+ * Rating component is a wrapper for one or more rating items that will be used to set a rating value.
```
Comment (sopranopillow) [inline packages/react-components/react-rating-preview/src/components/Rating/Rating.tsx:10]:
```suggestion
 * Rating is a wrapper for one or more rating items that will be used to set a rating value.
```

File: packages/react-components/react-rating-preview/src/components/RatingDisplay/RatingDisplay.tsx [hunk 2]
```diff
@@ -7,7 +7,8 @@ import type { RatingDisplayProps } from './RatingDisplay.types';
 import { useRatingDisplayContextValues } from './useRatingDisplayContextValues';
 
 /**
- * RatingDisplay component - TODO: add more docs
+ * RatingDisplay component is a wrapper for one or more rating items that will be used to display a rating value
```
Comment (sopranopillow) [inline packages/react-components/react-rating-preview/src/components/RatingDisplay/RatingDisplay.tsx:10]:
```suggestion
 * RatingDisplay is a wrapper for one or more rating items that will be used to display a rating value
```

File: packages/react-components/react-rating-preview/src/components/RatingItem/RatingItem.tsx [hunk 3]
```diff
@@ -6,7 +6,7 @@ import { useRatingItemStyles_unstable } from './useRatingItemStyles.styles';
 import type { RatingItemProps } from './RatingItem.types';
 
 /**
- * RatingItem component - TODO: add more docs
+ * RatingItem component is an item that will be used to set or display a rating value.
```
Comment (sopranopillow) [inline packages/react-components/react-rating-preview/src/components/RatingItem/RatingItem.tsx:9]:
```suggestion
 * RatingItem is an item that will be used to set or display a rating value.
```

**Review by sopranopillow — APPROVED** [review]

**Pull request merged**